open-api/rest-catalog-open-api.yaml

```diff
@@ -2893,6 +3003,37 @@ components:
           additionalProperties:
             type: string
 
+    AppendDataFileRequest:
+      type: object
+      required:
+        - data-files
+      properties:
+        identifier:
+          allOf:
+            - $ref: '#/components/schemas/TableIdentifier'
+          description: Table identifier to update; must be present for CommitTransactionRequest
+        accept-delay-ms:
+          type: integer
+          format: int64
+          description: Acceptable delay in milliseconds for processing the append request.
+          example: 300000
```

Comment:
what happened if REST catalog is busy and cannot fulfill such delay? Is this all or nothing append or best effort to append everything within delay-ms?